pkg/vulnsrc/julia/julia.go

```diff
@@ -0,0 +1,40 @@
+package julia
+
+import (
+	"path/filepath"
+
+	"github.com/aquasecurity/trivy-db/pkg/ecosystem"
+	"github.com/aquasecurity/trivy-db/pkg/types"
+	"github.com/aquasecurity/trivy-db/pkg/vulnsrc/osv"
+	"github.com/aquasecurity/trivy-db/pkg/vulnsrc/vulnerability"
+)
+
+const sourceID = vulnerability.Julia
+
+var osvDir = filepath.Join("vuln-list", "osv", "julia")
+
+type DatabaseSpecific struct {
+	URL string `json:"url"`
+}
```

Comment:
Do you plan to add these URLs into advisories?

If no - you can remove this struct.
If yes - you need to add TransformAdvisories function